storage/s3/s3.go

```diff
@@ -129,7 +129,8 @@ func New(cfg *Config, keys KeyProvider, logger zerolog.Logger) (Storage, error)
 
 	c, err := minio.NewWithRegion(cfg.Endpoint, cfg.AccessKey, cfg.AccessSecret, cfg.Secure, cfg.Region)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to initialize minio with region")
+		logger.Info().Err(err).Str("cmd", "s3::New").Msg("Failed to initialize minio with region")
```

Comment:
Don't we use the `errors.Wrap` to achieve the same logging functionality from a single point? (the error was also logged before)

Reply:
line 302 -> 317, 318 is actual fix for this thing = it was logged but not returned. In storage service and/or handlers the error is not logged. I guess it would make some sense to log it somewhere higher up but then we lose also information on exact place of where the error happened.